packages/features/landing_page_screen/lib/src/components/main_overflow_menu.dart

```diff
@@ -40,12 +42,27 @@ class _MainOverFlowMenuState extends ConsumerState<MainOverflowMenu> {
     );
   }
 
+  void _openStore() {
+    if (Platform.isAndroid || Platform.isIOS) {
+      final appId = Platform.isAndroid ? androidAppId : iOSAppId;
+      final url = Uri.parse(
+        Platform.isAndroid
+            ? 'market://details?id=$appId'
+            : 'https://apps.apple.com/at/app/pocket-code/id1117935892',
```

Comment:
Please use the variable and change its value.
iOSAppId would be obsolete right now.